edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/IConnectionHandler.cs

```diff
@@ -10,7 +10,11 @@ public interface IConnectionHandler
     {
         Task<IDeviceListener> GetDeviceListener();
 
-        Task<AmqpAuthentication> GetAmqpAuthentication();
+        //Task<AmqpAuthentication> GetAmqpAuthentication();
+
+        //IDeviceListener DeviceListener { get; }
+
+        //IAmqpAuthenticator AmqpAuthenticator { get; }
```

Comment:
Should these commented lines be removed?

Reply:
Ah, yes, let me do that